netbox/netbox/settings.py

```diff
@@ -469,7 +469,7 @@ def _setting(name, default=None):
 
 DEFAULT_AUTO_FIELD = 'django.db.models.BigAutoField'
 
-TEST_RUNNER = "django_rich.test.RichRunner"
+# TEST_RUNNER = "django_rich.test.RichRunner"
```

Comment:
Guessing this was intended to be a temporary change? (Happy to make the test runner configurable.)

Reply:
Yeah, I do not like the colored output. Makes it difficult to find the issue. I'll revert this.